content/blog/2021/07/2021-07-29-introducing-conventional-commits-plugin-for-jenkins.adoc

```diff
@@ -0,0 +1,121 @@
+---
+:layout: post
+:title: "Introducing the Conventional Commits Plugin for Jenins"
+:tags:
+- gsoc
+- gsoc2021
+- conventionalcommits
+- plugin
+:author: adi10hero
+:sig: gsoc
+opengraph:
+  image: /images/gsoc/2021/conventional-commits/conventionalCommitsPluginForJenkins.png
+links:
+  discourse: https://community.jenkins.io/t/introducing-the-conventional-commits-plugin/272
+---
+
+image:/images/gsoc/2021/conventional-commits/conventionalCommitsPluginForJenkins.png[GSoC, height=420, role=center, float=center]
+
+The conventional commits plugin is Google Summer of Code project. Special thanks to the mentors Gareth Evans, Kristin Whetstone, Olivier Vernin and Alan Buradezwic 
```

Comment:
Asciidoc is easier to code read when we use sentence per line.  That's different from markdown where sentence per line would mean every line has a line break.
Also corrected spelling and added a word
```suggestion
The conventional commits plugin is a Google Summer of Code project.
Special thanks to the mentors Gareth Evans, Kristin Whetstone, Olivier Vernin and Allan Burdajewicz.
```